docker/docker-compose.yml

```diff
@@ -27,8 +27,8 @@ services:
   # .Net Core API Immunization
   immunization-api-dev:
     image: dts-vc-immunization-api-dev
-    # environment:
-    #   DB_CONNECTION_STRING: "Host=db;Port=${DATABASE_PORT};Database=${DATABASE_NAME};Username=${DATABASE_USER};Password=${DATABASE_PASSWORD};"
+    environment:
+      DB_CONNECTION_STRING: "Host=db;Port=${DATABASE_PORT};Database=${DATABASE_NAME};Username=${DATABASE_USER};Password=${DATABASE_PASSWORD};"
```

Comment:
I would recommend/suggest having the environment variable `DATABASE_NAME` split into `ISSUER_DATABASE_NAME` and `IMMUNIZATION_DATABSE_NAME` (see the `manage` script to set it) and pass it as an environment variable to both services. This way the configuration can be injected easily at runtime - the connection string will remain the same, it's just the values of the variables that will be different for each service (in this case, the db service/port/user/pwd are the same, but the schema is different).

Reply:
Thanks @esune that makes sense